### ISSUE TITLE
Copy DASH content to the new ontology

### ISSUE THREAD

**Issue by bergos** [rdfjs/shacl-ui#12]
The new ontology will be based on [DASH](https://datashapes.org/) (talking about the content, not the name). There are two options for how this can be done:

1. Copy everything and use PR afterward to adapt it to our needs.
2. Merge feature by feature (classes and properties; also bundled) via PR into an empty document.

Please vote with :tada: for 1. or :rocket: for 2.

**Comment by HolgerKnublauch** [comment]
In case it's useful, I created a subset of dash as a PR. This would go into a folder called input where we could also collect other relevant input vocabularies. The real main vocab should of course be placed elsewhere.

https://github.com/rdfjs/shacl-ui/pull/13

**Comment by bergos** [comment]
Thanks a lot @HolgerKnublauch, and don't forget to vote: :tada:

I voted for 1. because it will speed up our work. Starting from scratch is better if there is a need to explore different ideas, but multiple parties already adapted DASH, so I don't see a risk from that perspective.

**Comment by HolgerKnublauch** [comment]
I don't know how to vote now, because the submission from my branch is basically the starting point where I think the group could start with, then evolve it, even beyond recognition if needed. So I wouldn't copy everything from DASH but exactly the provided subset and go from there.

**Comment by bergos** [comment]
@HolgerKnublauch OK, I see. I didn't want to get lost in details with the first commit. It's now "initial set", with the idea that it's more than a single feature, and separate PRs are used to discuss features in detail.

**Comment by silas-joekel** [comment]
I think we could also have a mix between the two options.

When looking at the PR @HolgerKnublauch created I identified these topics:

- editors/viewers + vocabulary
- shape lists (`dash:DateOrDateTime`, `dash:HTMLOrStringOrLangString`)
- property roles (`dash:LabelRole`, `dash:IconRole`)
- constraint components (`dash:IndexedConstraintComponent`)
- node shapes (`dash:ListNodeShape`, `dash:ListShape`)
- reification

I'd propose we start with the editors/viewers vocabulary and adapt it to our needs (workflow 1). For the other topics we should have separate discussions (workflow 2).

**Comment by HolgerKnublauch** [comment]
Yes, the DASH widgets are the main contribution. And there should be a third widget type to support entering filters for search forms.

The shape lists cover the common scenarios that properties need something like sh:datatype xsd:string or rdf:HTML or rdf:langString, so that not every ontology has to repeat that large blank node structure.

**Comment by tfrancart** [comment]
> I'd propose we start with the editors/viewers vocabulary and adapt it to our needs (workflow 1). For the other topics we should have separate discussions (workflow 2).

Agree with this
